cliquet/__init__.py

```diff
@@ -154,14 +156,16 @@ def includeme(config):
     handle_api_redirection(config)
     config.add_tween("cliquet.end_of_life_tween_factory")
 
-    storage = config.maybe_dotted(settings['cliquet.storage_backend'])
+    storage = config.maybe_dotted(settings.get('cliquet.storage_backend',
+                                               DEFAULT_STORAGE_BACKEND))
     config.registry.storage = storage.load_from_config(config)
 
-    session = config.maybe_dotted(settings['cliquet.session_backend'])
+    session = config.maybe_dotted(settings.get('cliquet.session_backend',
+                                               DEFAULT_SESSION_BACKEND))
     config.registry.session = session.load_from_config(config)
 
-    config.registry.project_name = settings['cliquet.project_name']
-    config.registry.project_docs = settings['cliquet.project_docs']
+    config.registry.project_name = settings.get('cliquet.project_name', '')
```

Comment:
Ah okay, I understand your proposal I think. Let me rephrase so I'm sure we're on the same page here.

Since we're already passing the version information to cliquet in `initialize_cliquet`, we should reuse this version to display it on the home page.

If that's it, I concur :)